### PR TITLE
[CVP-2063] Add rehearsal part for v4.6 &v 4.8

### PR DESCRIPTION
**Pull request by sonam1412**

### CODE REVIEW

File: ci-operator/config/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6.yaml [hunk 1]
```diff
@@ -11,11 +11,57 @@ resources:
...
+      OO_PACKAGE: e2e-test-operator
+      OO_TARGET_NAMESPACES: '!install'
+      PYXIS_URL: ""
+    workflow: optional-operators-cvp-common-claim
 - as: cvp-common-aws
   cron: '@yearly'
   steps:
     cluster_profile: aws-cpaas
     workflow: optional-operators-cvp-common-aws
+- as: cvp-common-powervs
```
Comment (dirgim) [inline ci-operator/config/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6.yaml:39]: The `powervs`, `probe` and `ibmz` job are out of scope for this ticket and the support for these jobs doesn't exist now - I propose removing this entire section down to line 64

File: ci-operator/config/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.8.yaml [hunk 2]
```diff
@@ -13,11 +11,44 @@ resources:
...
+      OO_PACKAGE: e2e-test-operator
+      OO_TARGET_NAMESPACES: '!install'
+      PYXIS_URL: ""
+    workflow: optional-operators-cvp-common-claim
 - as: cvp-common-aws
   cron: '@yearly'
   steps:
     cluster_profile: aws-cpaas
     workflow: optional-operators-cvp-common-aws
+- as: cvp-common-powervs
```
Comment (dirgim) [inline ci-operator/config/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.8.yaml:39]: The powervs and probe jobs are out of scope for this ticket and the v4.8 support for these jobs doesn't exist now - I propose removing this entire section down to line 51

File: ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6-periodics.yaml [hunk 3]
```diff
@@ -71,3 +71,226 @@ periodics:
...
+    - name: hive-hive-credentials
+      secret:
+        secretName: hive-hive-credentials
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
```
Comment (dirgim) [inline ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6-periodics.yaml:135]: The powervs, probe and ibmz jobs are out of scope for this ticket and the support for these jobs doesn't exist now - I propose removing this entire section down to line 296

File: ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6-periodics.yaml [hunk 4]
```diff
@@ -71,3 +71,226 @@ periodics:
...
+    volumes:
+    - name: ci-pull-credentials
+      secret:
+        secretName: ci-pull-credentials
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
```
Comment (dirgim) [inline ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6-periodics.yaml:296]: The powervs, probe and ibmz jobs are out of scope for this ticket and the support for these jobs doesn't exist now - I propose removing this entire section down to line 296

File: ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.8-periodics.yaml [hunk 5]
```diff
@@ -84,7 +83,175 @@ periodics:
...
+  cron: '@yearly'
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: ocp-4.8
+    org: redhat-openshift-ecosystem
+    repo: cvp
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
```
Comment (dirgim) [inline ci-operator/jobs/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.8-periodics.yaml:254]: The powervs, probe and ibmz jobs are out of scope for this ticket and the support for these jobs doesn't exist now - I propose removing this entire section down to line 254

**Review by dirgim — CHANGES_REQUESTED** [review]

**Review by jsztuka — APPROVED** [review]
/LGTM

**Review by dirgim — APPROVED** [review]

**Pull request merged**